pkg/server/build.go

```diff
@@ -212,18 +212,13 @@ func (s *Server) saveExamples(result *BuildResult, dir string, config *model.Con
 	return nil
 }
 
+// Load the Cog config
 func (s *Server) ReadConfig(dir string) (*model.Config, error) {
-	configRaw, err := os.ReadFile(filepath.Join(dir, global.ConfigFilename))
-	if err != nil {
-		return nil, fmt.Errorf("Failed to read %s: %w", global.ConfigFilename, err)
-	}
-	config, err := model.ConfigFromYAML(configRaw)
+	config, _, err := config.GetConfig("") // Read from cwd by default
```

Comment:
I think `dir` here is the temporary directory on the server with the config inside it, so this needs to read from there instead of the working directory.

Reply:
Fixed, thanks for pointing it out!